### PR TITLE
http-api: Resolve issue and comment authors

### PR DESCRIPTION
**Pull request by sebastinez**
Signed-off-by: Sebastian Martinez <me@sebastinez.dev>

### CODE REVIEW

File: http-api/src/issues.rs [hunk 1]
```diff
@@ -68,10 +77,13 @@ async fn issue_handler(
...
     let whoami = person::local(&*storage).map_err(Error::LocalIdentity)?;
     let issues = issue::Issues::new(whoami, &ctx.paths, &storage).map_err(Error::Issues)?;
-    let issue = issues
+    let mut issue = issues
         .get(&project, &issue_id)
         .map_err(Error::Issues)?
         .ok_or(Error::NotFound)?;
+    issue
+        .resolve(storage.as_ref())
+        .map_err(Error::IdentityResolveError)?;
```
Comment (cloudhead) [inline http-api/src/issues.rs:86]: Do we want to bail here? I'm not sure. We should still be able to render the issue without a resolved author I would think?

File: http-api/src/issues.rs [hunk 2]
```diff
@@ -53,7 +53,16 @@ async fn issues_handler(ctx: Context, project: Urn) -> Result<impl Reply, Reject
         .all(&project)
         .map_err(Error::Issues)?
         .into_iter()
-        .map(|(id, issue)| Cob::new(id, issue))
+        .map(|(id, mut issue)| {
+            if let Err(e) = issue
+                .resolve(storage.as_ref())
+                .map_err(Error::IdentityResolveError)
+            {
+                tracing::debug!("Failed to resolve issue author: {}", e);
```
Comment (cloudhead) [inline http-api/src/issues.rs:61]:
```suggestion
                tracing::error!("Failed to resolve issue author: {}", e);
```

Comment (cloudhead) [reply http-api/src/issues.rs:61]: Would be good for this to stick out in the logs, or maybe `warn` is enough.

Comment (cloudhead) [reply http-api/src/issues.rs:61]: Also might be handy to know the issue id in the log.

File: http-api/src/error.rs [hunk 3]
```diff
@@ -87,6 +87,10 @@ pub enum Error {
     /// An error occurred with env variables.
     #[error(transparent)]
     VarError(#[from] std::env::VarError),
+
+    /// An error occurred during the identity resolving.
+    #[error(transparent)]
+    IdentityResolveError(#[from] radicle_common::cobs::ResolveError),
```
Comment (cloudhead) [inline http-api/src/error.rs:93]: We usually avoid repeating `Error` in the variants, so I'd rename this to `IdentityResolve`. You could also fix the one above to just `Var` or `Env` maybe.

File: http-api/src/issues.rs (hunk 1 above)
Comment (sebastinez) [reply http-api/src/issues.rs:86]: No you're right, we should just warn about it, for debugging purposes

File: http-api/src/issues.rs (hunk 2 above)
Comment (sebastinez) [reply http-api/src/issues.rs:61]: Great, yeah that makes sense.

File: http-api/src/error.rs (hunk 3 above)
Comment (sebastinez) [reply http-api/src/error.rs]: Ok great, I also fixed the `SiweVerificationError` and the `SiweParseError`

**Pull request merged**